src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/EnhancedFurnaceListener.java

```diff
@@ -67,7 +69,15 @@ public void onItemSmelt(FurnaceSmeltEvent e) {
 
             if (state instanceof Furnace) {
                 FurnaceInventory inventory = ((Furnace) state).getInventory();
-                int amount = inventory.getSmelting().getType().toString().endsWith("_ORE") ? ((EnhancedFurnace) sfItem).getRandomOutputAmount() : 1;
+                Material smelting = inventory.getSmelting().getType();
+                boolean multiplier = SlimefunTag.ORES.isTagged(smelting);
+
+                // In 1.17, the raw ores should benefit from luck multiplier
+                if (SlimefunPlugin.getMinecraftVersion().isAtLeast(MinecraftVersion.MINECRAFT_1_17) && SlimefunTag.RAW_METALS.isTagged(smelting)) {
+                    multiplier = true;
+                }
```

Comment:
This can be simplified a lot.
In earlier minecraft versions, this tag is going to be empty anyway, so the version check is kinda redundant.

```suggestion
                boolean multiplier = SlimefunTag.ORES.isTagged(smelting) || SlimefunTag.RAW_METALS.isTagged(smelting);
```